### PR TITLE
French translation update for 1.1.30

### PR DESCRIPTION
**Pull request by trebmuh**
As discussed on IRC.

### CODE REVIEW

File: res/i18n/fr/labels.json [hunk 1]
```diff
@@ -203,6 +217,7 @@
 		"low_cut": "Coupure basse",
 		"middle": "Milieu",
 		"presence": "Présence",
+		"show_": "Affichage",
```
Comment (sadko4u) [inline res/i18n/fr/labels.json:220]: Here the underscore in "show_" means that the text should be shorened to about 4 symbols.
Is it possible to fix?

File: res/i18n/fr/actions.json [hunk 2]
```diff
@@ -61,6 +62,7 @@
 	"profile": "Profil",
 	
 	"reset": "Réinitialiser",
+	"run_stop": "Exécuter/arrêter",
```
Comment (sadko4u) [inline res/i18n/fr/actions.json:65]: Is it possible to shorten words to 3-4 symbols per each?

**Review by sadko4u — COMMENTED** [review]
There are few questions, please have a look

File: res/i18n/fr/labels.json (hunk 1 above)
Comment (trebmuh) [reply res/i18n/fr/labels.json]: done

File: res/i18n/fr/actions.json (hunk 2 above)
Comment (trebmuh) [reply res/i18n/fr/actions.json]: done

**Comment by sadko4u** [comment]
Available in 1.1.30

**Pull request merged**